### ISSUE TITLE
Aztec: Add accessibility labels to toolbar buttons

### ISSUE THREAD

**Issue by rachelmcr** [wordpress-mobile/WordPress-iOS#6945]
### Expected behavior

All of the toolbar buttons in Aztec should have accessibility labels to describe what the button is for.

### Actual behavior

Currently, none of the toolbar buttons in Aztec have accessibility labels. (Voiceover users only get the information "Button" when selecting a button in the toolbar.)
